test/integration/ads_integration_test.cc

```diff
@@ -489,5 +493,112 @@ TEST_P(AdsConfigIntegrationTest, EdsClusterWithAdsConfigSource) {
   ads_stream_->finishGrpcStream(Grpc::Status::Ok);
 }
 
+const std::string static_resources_config = R"EOF(
+dynamic_resources:
+  ads_config:
+    api_type: GRPC
+static_resources:
+  listeners:
+  - name: dummy_listener
+    address:
+      socket_address:
+        address: 127.0.0.1
```

Comment:
I'd suggest taking the existing bootstrap built by the config utility helper and just augmenting the static resources in `ConfigHelper::addConfigModifier`. That will reduce boilerplate here and make the test more robust as we evolve the integration test bootstrap.